cartography/intel/aws/resources.py

```diff
@@ -41,49 +41,51 @@
 from .ec2.volumes import sync_ebs_volumes
 from .ec2.vpc import sync_vpc
 from .ec2.vpc_peerings import sync_vpc_peerings
+from .wafv2 import sync_wafv2
 
 RESOURCE_FUNCTIONS: Dict = {
-    'iam': iam.sync,
-    's3': s3.sync,
-    'dynamodb': dynamodb.sync,
-    'ec2:launch_templates': sync_ec2_launch_templates,
-    'ec2:autoscalinggroup': sync_ec2_auto_scaling_groups,
-    # `ec2:instance` must be included before `ssm` and `ec2:images`,
-    # they rely on EC2Instance data provided by this module.
-    'ec2:instance': sync_ec2_instances,
-    'ec2:images': sync_ec2_images,
-    'ec2:keypair': sync_ec2_key_pairs,
-    'ec2:load_balancer': sync_load_balancers,
+    # 'iam': iam.sync,
```

Comment:
I think you meant to leave this change out of the PR.

Reply:
oops.

fixed